Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -87,9 +88,34 @@ protected string GetFullPath(string filePath)
                 return filePath;
             }
 
-            // For relative files, qualify them based on where the PowerShell session is.
-            string currentDirectory = SessionState.Path.CurrentLocation.Path;
-            return Path.Combine(currentDirectory, filePath);
+            ProviderInfo provider = null;
```

Comment:
For future programmers, it might be useful to explain how a cmdlet can return multiple resolved file paths. Is it in the case of ambiguities, such as when using case-sensitive file paths when PowerShell is assumed to be case-insensitive?